templates/broker/broker.conf.erb

```diff
@@ -38,7 +38,7 @@ DEFAULT_MAX_TRACKED_ADDTL_STORAGE_PER_GEAR="0"
 <% if scope.lookupvar('::openshift_origin::mongodb_replicasets') == true %>
 MONGO_HOST_PORT="<%= scope.lookupvar('::openshift_origin::mongodb_replicasets_members').join(',') %>"
 <% else %>
- MONGO_HOST_PORT="<%= scope.lookupvar('::openshift_origin::datastore_fqdn') %>:<%= scope.lookupvar('::openshift_origin::mongodb_port') %>"
+MONGO_HOST_PORT="<%= scope.lookupvar('::openshift_origin::datastore_fqdn') %>:<%= scope.lookupvar('::openshift_origin::mongodb_port') %>"
```

Comment:
See if you can get rid of this diff, I think the original had an extra space that you've removed. I wouldn't ask for this but I'm pretty sure you'll have to update things anyway due to the extra + below. Just trying to keep history as clean as possible.